manifests/service.pp

```diff
@@ -0,0 +1,40 @@
+# Class: shibboleth_idp::service
+#
+# Manages the Jetty process
+#
+# Sample Usage:
+#
+#    sometype { 'foo':
+#      notify => Class['shibboleth_idp::service'],
+#    }
+#
+#
+class shibboleth_idp::service {
+  # The base class must be included first because parameter defaults depend on it
+  if ! defined(Class['shibboleth_idp::params']) {
+    fail('You must include the shibboleth_idp::params class before using any shibboleth_idp defined resources')
+  }
```

Comment:
This if is probably not needed. If its private you can assume the params is included.